ddp-workspace/projects/ddp-sdk/src/lib/services/sessionMemento.service.ts

```diff
@@ -159,11 +159,7 @@ export class SessionMementoService implements OnDestroy {
         this.renewSubscription && this.renewSubscription.unsubscribe();
     }
 
-    private hasOnlyUserGuid(session: Session): boolean {
-        return !session.idToken && !!session.userGuid;
-    }
-
-    private isSessionExpired(): boolean {
+    public isSessionExpired(): boolean {
         const session = this.sessionSubject.value;
         if (session === null) {
```

Comment:
I think the brackets aren't balanced here, resulting in this always returning `false`?  Or am I misreading?

Reply:
Hm, I think brackets are balanced.
If the session observable is alive we will skip `return false;` and jump to `return new Date().getTime() > session.expiresAt;`

Reply:
Probably GitHub cut some lines in code review view and confused you... This method:
``` ts
    public isSessionExpired(): boolean {
        const session = this.sessionSubject.value;
        if (session === null) {
            return false;
        }

        return new Date().getTime() > session.expiresAt;
    }
```